src/passes/pass.cpp

```diff
@@ -356,6 +356,12 @@ void PassRegistry::registerPasses() {
   registerPass("reorder-functions",
                "sorts functions by access frequency",
                createReorderFunctionsPass);
+  registerPass("reorder-globals",
+               "sorts globals by access frequency",
+               createReorderGlobalsPass);
+  registerPass("reorder-globals-always",
```

Comment:
This will keep the pass from showing up in the `--help` output.
```suggestion
  registerTestPass("reorder-globals-always",
```